stackable-base/Dockerfile

```diff
@@ -4,7 +4,32 @@
 # https://catalog.redhat.com/software/containers/ubi9/ubi-minimal/615bd9b4075b022acc111bf5?image=6633b5136d81a5634a616c41&architecture=amd64
 
 # Manifest list digest because of multi architecture builds ( https://www.redhat.com/architect/pull-container-image#:~:text=A%20manifest%20list%20exists%20to,system%20on%20a%20specific%20architecture )
-FROM registry.access.redhat.com/ubi9/ubi-minimal@sha256:a7d837b00520a32502ada85ae339e33510cdfdbc8d2ddf460cc838e12ec5fa5a AS builder
+FROM registry.access.redhat.com/ubi9/ubi-minimal@sha256:a7d837b00520a32502ada85ae339e33510cdfdbc8d2ddf460cc838e12ec5fa5a AS product-utils-builder
+
+ENV CONFIG_UTILS_VERSION=0.1.0
+# This SHOULD to be kept in sync with operator-templating and other tools to reduce build times
+ENV RUST_DEFAULT_TOOLCHAIN_VERSION=1.78.0
+ENV CARGO_CYCLONEDX_CRATE_VERSION=0.5.3
```

Comment:
Thank you for updating the versions. Unfortunately 0.5 changes the CLI and we need to adapt. Can you keep it at the old version for now?

Reply:
auditable can stay updated

Reply:
I can downgrade as well, but I already adopted the CLI (nit sure if I did it correctly)

Reply:
On the new image the sbom still seems correct: `docker run -it --entrypoint bash docker.stackable.tech/stackable/stackable-base:1.0.0-stackable0.0.0-dev -c "cat /stackable/config-utils.cdx.xml | wc -l"`

Reply:
Ok, `cargo cyclonedx --output-pattern package --all --output-cdx` produced only 449 lines, so we have more than before. I leave it to you if we want to downgrade again or stick with 0.5.3

Reply:
yeah, sorry, I don't have time to look at those. Please revert the upgrade for now. There could be subtle changes in there.

Reply:
Sure, downgraded again